### PR TITLE
Meenu/fix: dmt5 banner fix

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   Fixed dmt5 banner

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_dherodmt5.js [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    min-height: 575px;
+    flex-direction: row-reverse;
+
+    @media ${device.desktopS} {
+        padding-left: 18%;
+    }
+    @media ${device.tabletL} {
+        flex-direction: column;
+        padding-left: 8%;
+    }
```
Comment (prince-deriv) [inline src/components/custom/_dherodmt5.js:26]: Can we avoid using percentage value?

File: src/components/custom/_dherodmt5.js [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+`
+const StyledHeaderTitle = styled.div`
+    color: var(--color-white);
+    display: flex;
+    margin-top: 0;
+    font-size: 64px;
+    font-weight: 700;
+    line-height: 80px;
+    @media ${device.tabletL} {
```
Comment (prince-deriv) [inline src/components/custom/_dherodmt5.js:50]: add space here

**Review by prince-deriv — APPROVED** [review]

File: src/components/custom/_dherodmt5.js (hunk 1 above)
Comment (meenakshi-deriv) [reply src/components/custom/_dherodmt5.js]: sure, i'll update

File: src/components/custom/_dhero-dmt5.js [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+        left: auto;
+        position: relative;
+        margin-bottom: 40px;
+    }
+`
+
+const LinkWrapper = styled.div`
+    display: flex;
+    margin-top: 32px;
+    @media (max-width: 1420px) {
```
Comment (michio-rework) [inline src/components/custom/_dhero-dmt5.js:99]: can we use theme's breakpoints here? maybe `labtopL` ?

File: src/components/custom/_dhero-dmt5.js [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+        width: 100%;
+        height: 100%;
+    }
+`
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 562px;
+    gap: 16px;
+
+    @media (max-width: 1370px) {
```
Comment (michio-rework) [inline src/components/custom/_dhero-dmt5.js:166]: we have to use theme's breakpoints as much as possible, can we use them here?

**Review by michio-rework — DISMISSED** [review]

**Review by prince-deriv — DISMISSED** [review]

**Review by prince-deriv — APPROVED** [review]

**Review by mohammad-rework — COMMENTED** [review]
Can we minify the svg file?

**Comment by meenakshi-deriv** [comment]
> Can we minify the svg file?

Hi @mohammad-rework , i have done optimization in another PR. And the file size is now reduced to 300Kb

**Pull request merged**